.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c
 osx_image: xcode10.1
-xcode_workspace: OktaAuth.xcworkspace
-xcode_scheme: OktaAuth iOS
-xcode_destination: platform=iOS Simulator,OS=11.0,name=iPhone 8
+xcode_workspace: OktaAuthSdk.xcworkspace
+xcode_scheme: OktaAuthNative iOS
+xcode_destination: platform=iOS Simulator,OS=12.1,name=iPhone 8
```

Comment:
Can we use `latest` instead of pinning to a direct version?

It would be beneficial to know when our tests fail on OS updates.